File: lib-model/src/main/java/com/minesec/msa/client/sdk/management/model/DeviceInfoResponseDto.java
package com.minesec.msa.client.sdk.management.model;

import com.minesec.msa.client.sdk.ResultDto;

import lombok.Getter;

/**
 * @author eric.song
 * @since 2023/4/20 11:17
 */
@Getter
public class DeviceInfoResponseDto extends ResultDto {

    private DeviceInfoData data;
}

File: lib-model/src/main/java/com/minesec/msa/client/sdk/management/model_k/DeviceInfoResponseDto_.kt
package com.minesec.msa.client.sdk.management.model_k
/**
 * @author eric.song
 * @since 2024/6/18 14:13
 */
data class DeviceInfoResponseDto_(val data: DeviceInfoData_, val success: Int, val message: String): ResultDto_(success, message)
